evennia/contrib/rpg/rpsystem/tests.py

```diff
@@ -338,6 +334,46 @@ def setUp(self):
         self.char1.swap_typeclass(rpsystem.ContribRPCharacter)
         self.char2.swap_typeclass(rpsystem.ContribRPCharacter)
 
+    def test_REMOVE_ME_PLEASE_DEAR_GOD_REMOVE_ME(self):
```

Comment:
**TODO: REMOVE THIS**

Just added so I could trigger the "normal" `search` method in tests for debugging.